irispy/spectrograph.py

```diff
@@ -229,17 +229,29 @@ def __init__(self, filenames, spectral_windows="All"):
 
     def __repr__(self):
         spectral_window = self.spectral_windows["name"][0]
-        spectral_windows_info = "".join(
-            ["\n    {0}\n        (raster axis: {1}, slit axis: {2}, spectral axis: {3})".format(
-            name, len(self.data[name].raster_axis), len(self.data[name].slit_axis),
```

Comment:
suppose someone studies only one raster_axis so he/she truncates by index then length of an integer gives an error.